src/briefcase/platforms/linux/system.py

```diff
@@ -0,0 +1,1003 @@
+            if rpmbuild_path.exists():
+                self.tools.shutil.rmtree(rpmbuild_path)
+
+            (rpmbuild_path / "BUILD").mkdir(parents=True)
+            (rpmbuild_path / "BUILDROOT").mkdir(parents=True)
+            (rpmbuild_path / "RPMS").mkdir(parents=True)
+            (rpmbuild_path / "SOURCES").mkdir(parents=True)
+            (rpmbuild_path / "SRPMS").mkdir(parents=True)
+            (rpmbuild_path / "SPECS").mkdir(parents=True)
+
+        # Add runtime package dependencies. App config has been finalized,
+        # so this will be the target-specific definition, if one exists.
+        system_runtime_requires = [
+            "python3",
+        ] + getattr(app, "system_runtime_requires", [])
+
+        # Write the spec file
+        with self.input.wait_bar("Write RPM spec file..."):
+            with (rpmbuild_path / "SPECS" / f"{app.app_name}.spec").open(
+                "w", encoding="utf-8"
+            ) as f:
+                f.write(
+                    "\n".join(
+                        [
+                            # By default, rpmbuild thinks all .py files are executable,
+                            # and if a .py doesn't have a shebang line, it will
+                            # tell you that it will remove the executable bit -
+                            # even if the executable bit isn't set.
+                            # We disable the processor that does this.
+                            "%undefine __brp_mangle_shebangs",
+                            # Base package metadata
+                            f"Name:           {app.app_name}",
+                            f"Version:        {app.version}",
+                            f"Release:        {getattr(app, 'revision', 1)}%{{?dist}}",
+                            f"Summary:        {app.description}",
+                            "",
+                            f"License:        {getattr(app, 'license', 'Unknown')}",
+                            f"URL:            {app.url}",
+                            "Source0:        %{name}-%{version}.tar.gz",
+                            "",
+                        ]
+                        + [
+                            f"Requires:       {requirement}"
+                            for requirement in system_runtime_requires
+                        ]
+                        + [
+                            "",
+                            f"ExclusiveArch:  {self.tools.host_arch}",
+                            "",
+                            "%description",
+                            app.long_description,
+                            "",
+                            # Disable all the auto-detection that tries to magically
+                            # determine requirements from the binaries
+                            f"%global __requires_exclude /usr/lib/{app.app_name}",
+                            f"%global __provides_exclude /usr/lib/{app.app_name}",
```

Comment:
Turns out this doesn't actually do what we want....`rpm` has a lot going on in these files...

The `__requires/provides_exclude` macros run against the auto-generated list of dependencies from the files being packaged. That's why it worked for me; I told it to ignore any dependencies it found matching `libQt*`.

However, our generalized solution is trying to prevent evaluating the files being packaged for dependencies at all. This is done with `__requires/provides_exclude_from`; I also implemented the macro for the `lib` directory since it can change for different platforms:
```suggestion
                            # Disable all the auto-detection that tries to magically
                            # determine requirements from the binaries
                            f"%global __requires_exclude_from ^%{{_libdir}}/{app.app_name}/.*$",
                            f"%global __provides_exclude_from ^%{{_libdir}}/{app.app_name}/.*$",
```